packages/codemods/README.md

```diff
@@ -28,6 +28,11 @@ node ./packages/codemods/bin/cli.js v3/dropdown path/to/some/glob/**/*.hbs
 * [v3/side-nav](transforms/v3/side-nav/README.md)
 <!--TRANSFORMS_END-->
 
+### v4
+<!--TRANSFORMS_START-->
```

Comment:
@alex-ju what happens to these automatic updates?

Reply:
when I generate a new transform it will get appended to this list – giving a way for people to discover them through our readme docs. not sure this answers your question though

Reply:
ok, so it's only on generation? in that case I should remove the start/end from the v3 block?

Reply:
yes, only on generation; we can remove it from the v3 block. it may still work but not sure how it behaves if there are two blocks (it may pick the latest), but safer to remove it from the old block